composer.json

```diff
@@ -14,8 +14,8 @@
     "require": {
         "php": ">=7.2",
         "guzzlehttp/guzzle": "^6.2",
-        "illuminate/notifications": "~5.5 || ^6.0 || ^7.0",
-        "illuminate/support": "~5.5 || ^6.0 || ^7.0"
+        "illuminate/notifications": "~5.5 || ^6.0 || ^7.0 || ^8.0",
```

Comment:
Maybe worth dropping 5.x since it's fully out of support now.

Reply:
I'll let @casperboone decide on that, if he agree's I can remove it.

Reply:
Let's drop that once we do a somewhat larger general refresh of the package :)